daemon/clients/base.py

```diff
@@ -108,6 +108,12 @@ async def list(self) -> Dict:
                     f'got response {response.status} while listing all {self._kind}s'
                 )
 
+    @if_alive
+    async def clear(self):
+        async with aiohttp.request(method='DELETE', url=f'{self.store_api}') as response:
+            if response.status == HTTPStatus.OK:
```

Comment:
It might be useful to log the error in the case `response.status` is not OK?

Reply:
yes i will add a else case so it logs the error.